src/App.js

```diff
@@ -1,23 +1,25 @@
-import logo from './logo.svg';
 import './App.css';
+import { BrowserRouter as Router, Route, NavLink, Redirect } from 'react-router-dom';
+import ScreenColor from './component/ScreenColor/ScreenColor.js';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+        <header>
+          <Route exact path="/">
```

Comment:
Best practice is to keep your BrowserRouter, Switch, and Routes all in the same file. For this app, I would suggest keeping them all in your App.js.